src/Microsoft.PowerShell.Commands.Management/resources/CmdletizationResources.resx

```diff
@@ -149,7 +149,7 @@
   </data>
   <data name="CimJob_SafeQueryDescription" xml:space="preserve">
     <value>CIM query for instances of the {0} class on the {1} CIM server: {2}</value>
-    <comment>{0} is a placeholder for a name of a CIM class.  Example: "Win32_Proccess".
+    <comment>{0} is a placeholder for a name of a CIM class.  Example: "Win32_Process".
```

Comment:
`CimJob_InvalidClassName` has:

> {0} is a placeholder for a name of a (potentially misspelled) CIM class.  Example: "Win32_Proccess".
> {1} is a placeholder for a server name.  Example: "localhost".

And there the goal seems to be to offer an actual misspelling (although, tbh, I'd suggest something that's more clearly a misspelling), everywhere else, I don't *think* that's the goal.